client/components/paginate/Thumbnail.tsx

```diff
@@ -1,8 +1,13 @@
 import React, { useEffect, forwardRef } from "react";
-import PropTypes from "prop-types";
 import { usePaginatorContext } from "../../context/PaginatorContext";
 
-const Thumbnail = forwardRef(({ scrollTo, selected, number }, ref) => {
+interface ThumbnailProps {
+    scrollTo: void;
```

Comment:
Another void function. Apologies for raising this issue repeatedly if I'm wrong about it, but I think a change may be justified.

Reply:
Made change.